contributors/devel/sig-node/triage.md

```diff
@@ -0,0 +1,153 @@
+and deprecations require special review; those labels may be mistakenly
+applied, so check over the PR to make sure they're accurate.
+
+Once you've quickly looked over a PR, applied the appropriate labels, and it
+looks ready to proceed to review (i.e. it doesn't have any labels that would
+mean it's waiting on more work from the author), you can mark the PR as triaged
+with `/triage accepted`.
+
+Only [Kubernetes org members] can add this label.
+
+## Waiting on Author
+
+This column means that the PR is waiting on some action from the author. A
+reviewer may have requested changes, or a PR may have one of the following
+do-not-merge labels:
+
+- **do-not-merge/hold:** usually set by a reviewer
+- **do-not-merge/work-in-progress:** usually set by an author
+- **do-not-merge/release-note-needed:** needs a release note to be added by the
+  author, [Kubernetes org members] can override with `/release-note-none` if
+  not required
+- **do-not-merge/contains-merge-commits:** PR needs to be rebased
+- **needs-rebase:** PR needs to be rebased
+
+PRs that do not have any of the above labeled in this column should be
+evaluated occasionally to see if they are ready for review.
+
+## Waiting on Reviewer
+
+This PR needs review! If you're not sure how to review a PR, start by
+familiarizing yourself with the Kubernetes [pull request guidelines] and
+[review guidelines].
+
+PRs in this column must have the following labels set:
+
+- triage-accepted
+- priority
+- kind
+
+Only [Kubernetes org members] can add an `/lgtm`.
+
+If you want to become an official Node reviewer, you should read through the
+[reviewer responsibilities and requirements].
+
+TODO: Add some node-specific stuff here.
+
+[pull request guidelines]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md
+[review guidelines]: https://github.com/kubernetes/community/blob/master/contributors/guide/review-guidelines.md
+[reviewer responsibilities and requirements]: https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer
+
+## Waiting on Approver
+
+These PRs are waiting on an `approved` label that can only be provided by
+approvers. The bot will always tell you whose approval is needed on which
+directories.
+
+PRs in this column must have the following labels set:
+
+- lgtm
+- triage-accepted
+- priority
+- kind
+
+Check for the bot's comment to see which files still need approvers from the
+appropriate OWNERS. If the PR already has an approval for the node components
+(commonly, anything in `./pkg/kubelet/*`), you can mark the PR as Done manually
+while waiting on other approvers.
+
+Anyone can `/approve` a PR, but the bot will only pay attention to the
+approvals from [Kubernetes approvers].
+
+[Kubernetes approvers]: https://github.com/kubernetes/community/blob/master/community-membership.md#approver
+
+## Done
+
+This column is for closed and merged PRs. Huzzah!
```

Comment:
will this column contain PRs that are waiting for approval from other OWNERS, API review, or release managers for cherry-picks?